.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Avoid having multiple commit in single branch, always use git commit --amend in order to update the current commit